### PR TITLE
Add display_result to new RDB output (IOW-620)

### PR DESCRIPTION
**Pull request by kkehl-usgs**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [x] Update the changelog appropriately

Title
-----------
The discrete_ground_water_aqts table replaces two columns with a new column 'display_result'.  Use the new column to populate rdb.

Description
-----------

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

**Review by ssoper-usgs — APPROVED** [review]
Looks good!

File: src/main/java/gov/usgs/wma/waterdata/groundwater/RdbWriter.java [hunk 1]
```diff
@@ -77,10 +77,25 @@ public RdbWriter writeRow(DiscreteGroundWater dgw) {
...
+			writeValue(  1, "L"); // entry code for below Land
+			writeValue( 10, "");
+			writeValue(  8, "");
+		}*/
 		writeValue(  7, "");
-		writeValue(  1, "");
+		writeValue(  1, "S");
 		writeValue( 10, dgw.verticalDatumCode);
-		writeValue(  8, "");
+		writeValue(  8, dgw.displayResult);
```
Comment (duselmann) [inline src/main/java/gov/usgs/wma/waterdata/groundwater/RdbWriter.java]: Is displayResult similar to levelFeetAboveVerticalDatum or levelFeetBelowLandSurface ?
Maybe we should just have placed that value in the filed that it represents and let this go unchanged?
no sure.

Comment (jkreft-usgs) [reply src/main/java/gov/usgs/wma/waterdata/groundwater/RdbWriter.java]: yes, we need to map display result to `levelFeetAboveVerticalDatum` and `levelFeetBelowLandSurface` based on parameter code

Comment (duselmann) [reply src/main/java/gov/usgs/wma/waterdata/groundwater/RdbWriter.java]: @kkehl-usgs we can work together on this if you like.

Comment (kkehl-usgs) [reply src/main/java/gov/usgs/wma/waterdata/groundwater/RdbWriter.java]: Updated with changes based on pair session.

**Review by duselmann — CHANGES_REQUESTED** [review]
I do not think that swapping these two is appropriate. Oh, they are swapped back... I did not see the commits in orders and the conditional was not in the snip.

**Comment by kkehl-usgs** [comment]
> This conditional say "If A is empty then write A" rather than "If A is empty then write B"

I assume you are referring to the penultimate version.   The latest version has this:

if (StringUtils.isEmpty(dgw.levelFeetBelowLandSurface)) {
			writeValue(  7, "");
			writeValue(  1, "S"); // entry code for above Sea
			writeValue( 10, dgw.verticalDatumCode);
			writeValue(  8, dgw.levelFeetAboveVerticalDatum);
		} else {
			writeValue(  7, dgw.levelFeetBelowLandSurface);
			writeValue(  1, "L"); // entry code for below Land
			writeValue( 10, "");
			writeValue(  8, "");
		}

**Pull request closed without merging**